pkg/server/home_workspaces.go

```diff
@@ -83,7 +85,22 @@ func init() {
 // - bucketSize is the number of chars comprising each bucket.
 //
 // Bucket workspace names are calculated based on the user name hash.
-func WithHomeWorkspaces(apiHandler http.Handler, a authorizer.Authorizer, kubeClusterClient kubernetes.ClusterInterface, kcpClusterClient kcpclient.ClusterInterface, kubeSharedInformerFactory coreexternalversions.SharedInformerFactory, kcpSharedInformerFactory kcpexternalversions.SharedInformerFactory, externalHost string, creationDelaySeconds int, homePrefix logicalcluster.Name, bucketLevels, bucketSize int) http.Handler {
+func WithHomeWorkspaces(
+	apiHandler http.Handler,
+	a authorizer.Authorizer,
+	kubeClusterClient kubernetes.ClusterInterface,
+	kcpClusterClient kcpclient.ClusterInterface,
+	kubeSharedInformerFactory coreexternalversions.SharedInformerFactory,
+	kcpSharedInformerFactory kcpexternalversions.SharedInformerFactory,
+	externalHost string,
+	creationDelaySeconds int,
+	homePrefix logicalcluster.Name,
+	bucketLevels,
+	bucketSize int,
```

Comment:
nit: not all parameters are documented, also let's document the rationale of limiting bucketLevels/bucketSize.

Reply:
@davidfestal follow-up

Reply:
added to https://github.com/kcp-dev/kcp/issues/1069